### ISSUE TITLE
Floating gallbladder

### ISSUE THREAD

**Issue by MickeySegal** [obophenotype/human-phenotype-ontology#10048]
**Preferred term label:**
Floating gallbladder

**Synonyms**

**Definition (free text, please give PubMed ID)**
https://www.ncbi.nlm.nih.gov/medgen/C0266254

**Parent term (use hpo.jax.org/app)**

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
Gallbladder volvulus

**Your nano-attribution (ORCID)**

**Comment by pnrobinson** [comment]
@MickeySegal 
added as
HP:0020227
